styles/layout.scss

```diff
@@ -0,0 +1,196 @@
+.shop-item {
+  border: $c-primary solid 2px;
+
+}
+
+.shop-item__details {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  color: $c-primary;
+  text-align: center;
+
+}
+
+.shop-item__actions {
+  display: flex;
+  flex-direction: row;
+  // position: absolute;
+}
+
+.shop-item__frame {
+  margin: $c-gutter;
+}
+
+.shop-item__actions-btn {
+  display: flex;
+  justify-content: center;
+  flex-grow: 1;
+
+  border: $c-primary 1px solid;
+  border-bottom: none;
+  border-left: none;
+
+  &:last-of-type {
+    border-right: none;
+  }
+
+  padding: 1rem;
+  background-color: $c-white;
+  color: $c-primary;
+
+  &:hover {
+
+    background-color: $c-primary;
+    color: $c-white;
+
+  }
+
+}
+
+.shop-item__actions-btn {
+  position: relative;
+}
+
+.shop-menu {
+  &__tooltip {
+    display: none;
+    position: absolute;
+
+    flex-direction: column;
+    bottom: 2rem;
+
+    &:hover {
+
+      background: $c-white;
+      color: $c-primary;
+    }
+  }
+
+  &:hover {
+    color: $c-white;
+
+    .shop-menu__tooltip {
+      display: flex;
+      background: $c-white;
+      color: $c-primary;
+
+    }
+  }
+}
+
+.shop-menu__tooltip-item {
+
+  padding: 1rem;
+  border: $c-primary 2px solid;
```

Comment:
Look at putting borders only on some edges and using the `:first-of-type` or `:last-of-type` selector to fix the current double borders around your items.

<img width="113" alt="screen shot 2016-10-12 at 2 40 32 pm" src="https://cloud.githubusercontent.com/assets/2532004/19325059/0e797ec4-908a-11e6-971c-0c9d0794cfed.png">